lib/stormpath-sdk/resource/directory.rb

```diff
@@ -28,6 +28,7 @@ class Stormpath::Resource::Directory < Stormpath::Resource::Instance
   has_one :custom_data
   has_one :password_policy
   has_one :account_creation_policy
+  has_one :account_schema, class_name: :schema, href: '/accountSchema'
```

Comment:
By looking at the account_creation_policy.rb file and the relationship with the verification_email_templates, I'd say that you don't need this href.